local_exception_logger/__init__.py,sha256=IWhiAf-QvTYtJ54c_12qvubFnf8DEyzBCLlFY_e4s4g,39
local_exception_logger/core.py,sha256=3J1kuKVew1ejcjNcK_G202lL5NiV1op3b7T5aGQg6F4,2997
local_exception_logger-0.1.1.dist-info/METADATA,sha256=jztRK-4g8h3mzw9g8mIhuR95ljoJJaD8WYDRRKDQWO4,311
local_exception_logger-0.1.1.dist-info/WHEEL,sha256=pxyMxgL8-pra_rKaQ4drOZAegBVuX-G_4nRHjjgWbmo,91
local_exception_logger-0.1.1.dist-info/top_level.txt,sha256=yZhv-54mprhAU4a6qZl2QU1T3FVisnzPzpHFcAEbqXc,23
local_exception_logger-0.1.1.dist-info/RECORD,,
